pkg/performanceprofile/controller/performanceprofile/components/profile/profile.go

```diff
@@ -56,7 +56,20 @@ func IsPaused(profile *performancev2.PerformanceProfile) bool {
 	return false
 }
 
-// IsRpsEnabled or not RPS mask should be set for all physical net devices
+// IsPyhsicalRpsEnabled checks if RPS mask should be set for all physical net devices
```

Comment:
```suggestion
// IsPhysicalRpsEnabled checks if RPS mask should be set for all physical net devices
```

Reply:
I can't really spot the difference here. 
I assume you meant: change  IsPhysicalRpsEnabled  to IsVirtualRpsEnabled ?

Reply:
IsP*yh*sical vs IsP*hy*sical - letter swap.